main/FatesRestartInterfaceMod.F90

```diff
@@ -72,38 +87,21 @@ module FatesRestartInterfaceMod
   integer, private :: ir_seedrainflux_si
   integer, private :: ir_trunk_product_si
   integer, private :: ir_ncohort_pa
-  integer, private :: ir_bsw_co
-  integer, private :: ir_bdead_co
-  integer, private :: ir_bleaf_co
-  integer, private :: ir_broot_co
-  integer, private :: ir_bstore_co
+
   integer, private :: ir_canopy_layer_co
   integer, private :: ir_canopy_layer_yesterday_co
   integer, private :: ir_canopy_trim_co
   integer, private :: ir_dbh_co
   integer, private :: ir_height_co
   integer, private :: ir_laimemory_co
-  integer, private :: ir_leaf_md_co
```

Comment:
guessing these are being deleted since PARTEH unequivocally removed them from the state vector?